packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.html

```diff
@@ -42,15 +44,26 @@
           <div data-cy="generated-answer__answer" class={generatedAnswerClass}>
             {answer}
           </div>
-          <template lwc:if={hasCitations}>
-            <div class="slds-var-m-top_small">
-              <c-quantic-source-citations
-                data-cy="generated-answer__citations"
-                citations={citations}
-                citation-click-handler={handleCitationClick}
-              ></c-quantic-source-citations>
-            </div>
-          </template>
+          <div data-cy="generated-answer__footer" class={generatedAnswerFooterCssClass}>
+            <template lwc:if={hasCitations}>
+              <div class="slds-var-m-top_small">
+                <c-quantic-source-citations
+                  data-cy="generated-answer__citations"
+                  citations={citations}
+                  citation-click-handler={handleCitationClick}
+                ></c-quantic-source-citations>
+              </div>
+            </template>
+            <template lwc:if={shouldDisplayActions}>
```

Comment:
yes it is normal, the actions we have in the generated answer component are the feedback buttons and the rephrase buttons, and both need to be displayed only  after getting all the answer.

Since these two elements, the feedback and the rephrase, are displayed in different locations in the component, we need to have this condition in two multiple locations in order to display the feedback and the rephrase buttons only when needed.

Reply:
oh sorry I missed the closing tag, I thought the first `shouldDisplayActions` was around the 2nd `shouldDisplayActions` that's why I was confused